Formula/m/mariadb.rb

```diff
@@ -94,9 +100,6 @@ def install
       args << "-DCONNECT_WITH_JDBC=OFF"
     end
 
-    # Disable RocksDB on Apple Silicon (currently not supported)
-    args << "-DPLUGIN_ROCKSDB=NO" if Hardware::CPU.arm?
-
```

Comment:
Same logic exists in CMake script - https://github.com/MariaDB/server/blob/mariadb-11.5.2/storage/rocksdb/CMakeLists.txt#L72-L74